BeeKit/UI/BSButton.swift

```diff
@@ -13,23 +13,52 @@ public class BSButton : UIButton {
     
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        
+        registerForTraitChanges(
+            [UITraitUserInterfaceLevel.self]) {
```

Comment:
Are you confident this is the correct trait to watch? https://developer.apple.com/documentation/uikit/uitraitcollection/3238085-userinterfacelevel doesn't make it sound particularly relevant to buttons.

Reply:
UITraitUserInterfaceStyle looks to be more on point.